mastodon/src/main/java/org/joinmastodon/android/fragments/settings/BehaviourFragment.java

```diff
@@ -63,6 +64,10 @@ public void addItems(ArrayList<Item> items) {
             GlobalUserPreferences.save();
             needAppRestart=true;
         }));
+        items.add(new SwitchItem(R.string.mo_haptic_feedback, R.string.mo_setting_haptic_feedback, R.drawable.ic_fluent_phone_vibrate_24_filled, GlobalUserPreferences.hapticFeedback, i -> {
+            GlobalUserPreferences.hapticFeedback = i.checked;
+            GlobalUserPreferences.save();
+        }, Build.VERSION.SDK_INT >= Build.VERSION_CODES.R));
```

Comment:
I think this would fit better in the `Timeline` settings, as most other post settings are already there (e.g. showing interaction counts, etc...).

Reply:
I think I disagree, as this is more of a behavior thing, just like the "Confirm boost" setting, which could be either at the timelines section or on the behavior section. I think I prefer it on the Behavior settings

Reply:
I definitely see your point, and the timelines page, could probably be resolved and split into Behavior and Appearance pages. When I did the settings redesign, I moved all items, which have a direct effect on either a timeline or a thread view/ post, into the timeline page. From my point of view, the "Confirm boost" is an action the is not directly tied to the timelines, whereas the haptic feedback directly changes the interacts with a status, which is why it should be moved into the timeline page.